oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtAuthenticationProvider.java

```diff
@@ -107,6 +110,11 @@ public boolean supports(Class<?> authentication) {
 		return BearerTokenAuthenticationToken.class.isAssignableFrom(authentication);
 	}
 
+	public void setJwtConverter(Converter<Jwt, ? extends AbstractAuthenticationToken> jwtConverter) {
```

Comment:
Same as comment in DSL, we should consider maybe changing this to `setAuthenticationConverter()`